.github/stale.yml

```
@@ -12,22 +12,16 @@ staleLabel: "2.status: stale"
 markComment: >
```

I think changing this should fix it?

```suggestion
markComment: |
```